ga4gh/datamodel/variants.py

```diff
@@ -415,6 +436,27 @@ def convertVariant(self, record, callSetIds):
             sampleIterator += 1  # REMOVAL
         return variant
 
+    def getVariant(self, compoundId):
+        if compoundId.chrom in self._chromFileMap:
+            varFileName = self._chromFileMap[compoundId.chrom]
+        else:
+            raise exceptions.ObjectNotFoundException(compoundId)
+        pos = int(compoundId.pos)
```

Comment:
Why are we using pos here? Is this equal to the ``start`` value (0 based)? If not, we should just change the ``variant.id`` definition in convertVariant as indicated in the comments there.